fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -96,6 +96,20 @@
      */
     boolean isMemento();
 
+    /**
+     * Returns true if this resource is a NonRdfSourceDescription.
+     *
+     * @return true if the resource is a NonRdfSourceDescription.
+     */
+    boolean isDescription();
```

Comment:
I think this can be removed and just use `.getDescribedResource()` if you are unsure.

Reply:
removed.